The macos runner failed with the following error while installing the dependencies from homebrew:
```
==> sbt
You can use $SBT_OPTS to pass additional JVM options to sbt.
Project specific options should be placed in .sbtopts in the root of your project.
Global settings should be placed in /usr/local/etc/sbtopts

The installation does not include `sbtn`.
Error: Process completed with exit code 1.
```

I don't know what that means.

--- a/.github/workflows/macos.yaml
+++ b/.github/workflows/macos.yaml
@@ -35,7 +35,7 @@ jobs:
         # re-installed.
         run: |
           brew update
-          brew install --overwrite python@3.10 python@3.11
+          brew install --overwrite python@3.10 python@3.11 python@3.12
           brew reinstall gcc
           brew install autoconf automake ccache cmake gmp lapack libomp mpfr openblas
 

